pallets/automation-time/src/lib.rs

```diff
@@ -411,6 +420,10 @@ pub mod pallet {
 				Err(<Error<T>>::PastTime)?;
 			}
 
+			if scheduled_time >= current_time_slot + T::MaxScheduleSeconds::get() {
```

Comment:
just greater than rather than >=, i think? at least according to this comment: /// - Not be more than MaxScheduleSeconds out